docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -3239,72 +3239,72 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0153E":"(%s) is not a supported platform for %s. Skipping (%s)... Supported: %s
+### ZWED0153E":"(_operatingSystemName_) is not a supported platform for _componentName_. Skipping (_pluginid_)... Supported: _requiredOperatingSystem_
```

Comment:
These error messages are formatted incorrectly relative to previous PR's. We have been doing

 ie.
 
![image](https://user-images.githubusercontent.com/20528015/209060746-19fbb557-2dc1-4140-b1a0-ffe05154f83e.png)

Reply:
My understanding was to just replace string substitutions with more clear variable names, I don't recall discussing the change you mentioned. The reason after this line also provides the user with a further explanation.

Also if you scroll up in the same file you can find plenty of other messages that use the same formatting as this PR. Personally I don't find the difference in formatting/fillibustering to provide any additional clarity to the user.